docs/Developers/Git_Basics.rst

```diff
@@ -163,6 +163,19 @@ Now that your message is written, you will have to save the file. Press escape
 to exit insert mode, and save the file (in Vim that is being done by pressing
 shift + Z twice).
 
+Run Coala
+------------------
+
+Now you can check if your commit messages and the code formattings
+conform with community guidelines.
```

Comment:
Add `the` between `with` and `community` -> `conform with the community guidelines`